src/ingest-pipeline/misc/tools/schema_tester.py

```diff
@@ -0,0 +1,98 @@
+#! /usr/bin/env python
+
+#
+# This is a heavily modified version of the schema_tester.py routine from
+# pyRHEA, https://github.com/PSC-PublicHealth/pyrhea
+
+import sys
+from os import walk
+from os.path import (
+    basename, dirname, relpath, split, join, getsize,
+    realpath, exists, isdir
+)
+import optparse
+import yaml
+import json
+from typing import (
+    Any, Dict, List, Union
+)
+
+from jsonschema.exceptions import (
+    ValidationError,
+    SchemaError,
+    UnknownType,
+    UndefinedTypeCheck
+)
+from hubmap_commons.schema_tools import (
+    check_json_matches_schema
+)
+
+JSONType = Union[str, int, float, bool, None, Dict[str, Any], List[Any]]
+
+SCHEMA_BASE_PATH = join(dirname(dirname(dirname(realpath(__file__)))),
+                        'schemata')
+SCHEMA_BASE_URI = 'http://schemata.hubmapconsortium.org/'
```

Comment:
Is this in use? I haven't been able to reach such url.

Reply:
This is a strange bit- it's syntactically necessary to include a URL in the jsonschema, but it doesn't have to be a *real* URL, just a unique one.  I think it's serving as an anchor for parsing the rest of the schema reference.  But no, there is no such URL in real life.